src/service.rs

```diff
@@ -135,6 +134,12 @@ impl Service for AnchoringService {
                     panic!("A critical error occured: {}", e);
                 }
                 error!("An error in handler occured: {}", e);
+                if let Some(sink) = handler.errors_sink.as_ref() {
+                    let res = sink.send(e);
```

Comment:
Nitpick: seems like introducing `res` variable is unnecessary here. Also, could it be possible to move `if let HandlerError::IncorrectLect { .. } = e` condition into the `match` operator? Imo, `if let` operators that do not actually destructure stuff look slightly weird.

Reply:
Fixed